tests/qunit/js/editor-test.js

```diff
@@ -1,22 +1,21 @@
-import EditorBase from './../../../assets/dev/js/editor/editor-base';
+import EditorBase from 'elementor-editor/editor-base';
+import EditorElementor from 'elementor-editor/editor-elementor';
 
-export default class EditorTest extends EditorBase {
```

Comment:
On unit test the `EditorBase` should be used, without the UI from the real editor.